assignments/assignment02.rb

```diff
@@ -26,10 +32,21 @@ def to_sentence(ary)
 # implement methods "mean", "median" on Array of numbers
 def mean(ary)
   # your implementation here
+  ary.reduce(:+) / ary.length
 end
 
 def median(ary)
   # your implementation here
+  # the array has to be sorted for the algorithm to work
+  sorted_ary = ary.sort()
```

Comment:
in idiomatic ruby, you would not include the parens on the method call, instead you would just write:  sorted_ary = ary.sort